modules/dnn/include/opencv2/dnn/dnn.hpp

```diff
@@ -1135,6 +1147,76 @@ CV__DNN_INLINE_NS_BEGIN
                                    const Scalar& mean = Scalar(), bool swapRB=false, bool crop=false,
                                    int ddepth=CV_32F);
 
+    /**
+     * @brief Enum of image processing mode.
+     * @see ImagePParam
+     */
+    enum ImageProcessMode
```

Comment:
ImageProcessMode is very generic name. What about "ImagePaddingMode"?

Reply:
Fixed